pkgs/by-name/ru/runpodctl/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "runpodctl";
+  version = "1.14.2";
+
+  src = fetchFromGitHub {
+    owner = "runpod";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-eemg8W8kTjT9k3TKi+pqRisoxD+7KjetZoWFWbpHFV8=";
+  };
+
+  vendorHash = "sha256-9UkHYeDTZlEKEBfazj7N7QJrWcvVi5StxtYE70OF8tw=";
+
+  postInstall = ''
+    rm $out/bin/docs # remove the docs binary
+    mv $out/bin/cli $out/bin/runpodctl
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/runpod/runpodctl/";
```

```suggestion
    homepage = "https://github.com/runpod/runpodctl";
```
Maybe it would be better to use their official website?

---

i can link to https://docs.runpod.io/references/runpodctl/ but not sure how likely the url is to stay the same, github less likely to change?